xsct.c

```diff
@@ -0,0 +1,234 @@
+/*
+ * xsct - X11 set color temperature
+ *
+ * Compile the code using the following command:
+ * cc -std=c99 -O2 -I /usr/X11R6/include sct.c -o xsct -L /usr/X11R6/lib -lX11 -lXrandr
```

Comment:
can be deleted in favor of the `gcc` line below

Reply:
Resolve https://github.com/faf0/sct/pull/10/commits/8774dceb71da59fd952f6209d3a0d63496776fd3